main/lsp/LSPTypechecker.cc

```diff
@@ -22,7 +23,24 @@
 
 namespace sorbet::realmain::lsp {
 using namespace std;
+class ErrorCollector : public core::ErrorFlusher {
```

Comment:
(Minor nit) Can you put this into the `namespace {` below? That's an anonymous namespace that tells the compiler "this thing is only ever accessed in this file".